cfp.html

```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a non-profit event.</li>
                 <li>All talks will have to adhere to our <a href="/coc">Code of Conduct</a>.</li>
+                <li>We are fully committed to have a lineup as diverse as we can.</li>
```

Comment:
Maybe here link to the process we're doing in the CFP (or just the text "See our CFP process guidelines for more info")